python/datasources/acs_health_insurance.py

```diff
@@ -0,0 +1,657 @@
+    "_040E": meta(Sex.FEMALE, 26, 34),
+    "_041E": meta(Sex.FEMALE, 26, 34, HealthInsurancePopulation.WITH),
+    "_042E": meta(Sex.FEMALE, 26, 34, HealthInsurancePopulation.WITHOUT),
+    "_043E": meta(Sex.FEMALE, 35, 44),
+    "_044E": meta(Sex.FEMALE, 35, 44, HealthInsurancePopulation.WITH),
+    "_045E": meta(Sex.FEMALE, 35, 44, HealthInsurancePopulation.WITHOUT),
+    "_046E": meta(Sex.FEMALE, 45, 54),
+    "_047E": meta(Sex.FEMALE, 45, 54, HealthInsurancePopulation.WITH),
+    "_048E": meta(Sex.FEMALE, 45, 54, HealthInsurancePopulation.WITHOUT),
+    "_049E": meta(Sex.FEMALE, 55, 64),
+    "_050E": meta(Sex.FEMALE, 55, 64, HealthInsurancePopulation.WITH),
+    "_051E": meta(Sex.FEMALE, 55, 64, HealthInsurancePopulation.WITHOUT),
+    "_052E": meta(Sex.FEMALE, 65, 74),
+    "_053E": meta(Sex.FEMALE, 65, 74, HealthInsurancePopulation.WITH),
+    "_054E": meta(Sex.FEMALE, 65, 74, HealthInsurancePopulation.WITHOUT),
+    "_055E": meta(Sex.FEMALE, 65, None),
+    "_056E": meta(Sex.FEMALE, 65, None, HealthInsurancePopulation.WITH),
+    "_057E": meta(Sex.FEMALE, 65, None, HealthInsurancePopulation.WITHOUT),
+}
+
+# Given the Prefix: eg.
+# {"C27001A":{MetadataKey.RACE: Race.WHITE_ALONE}}
+# will combine with the provided suffixes eg.
+# {
+#   "_002E":meta(None,0, 19),
+#   "_003E":meta(None,0, 19,HealthInsurancePopulation.WITH)
+#   ...
+# }
+# to create the ACS variable params.
+# ?for=state&get=C27001A_002E,C27001A_003E...
+
+
+def format_params(prefixes, suffixes, isCounty=False):
+    groups = []
+    for prefix in prefixes:
+        for suffix in suffixes:
+            groups.append(prefix+suffix)
+    vars = ','.join(groups)
+
+    return {'for': 'county' if isCounty else 'state', "get": vars}
+
+# Helper to log response codes from ACS calls
+
+
+def log_response(resp):
+    print("-"*50)
+    print(f'Get Request ({resp.status_code}): {resp.url}')
+    print("-"*50)
+
+
+class AcsHealhInsuranceIngestor:
+
+    # Initialize variables in class instance, also merge all metadata so that lookup of the
+    # prefix, suffix combos can return the entire metadata
+    def __init__(self, base):
+        self.baseUrl = base
+        self.data = {}
+        self.metadata = {}
+        self.buildMetadataList()
+
+    # Gets standardized filename
+    def getFilename(self, sex, race, isCounty):
+        if(race is not None):
+            return "HEALTH_INSURANCE_BY_RACE_{0}_{1}.json".format("STATE" if isCounty else "COUNTY", race)
+        else:
+            return "HEALTH_INSURANCE_BY_SEX_{0}_{1}.json".format("STATE" if isCounty else "COUNTY", sex)
+
+    # Method to output <Filename.csv>.  Used for debugging purposes.
+    def write_local_files_debug(self):
+        self.getStateFipsMap()
+        self.getCountyFipsMap()
+        self.getHealthInsuranceDataBySex()
+        self.getHealthInsuranceDataByRace()
+        self.splitDataFrames()
+
+        with open('total_health_insurance.json', 'w') as f:
+            print(json.dumps(
```

Comment:
nit: i think it's more idiomatic to do f.write(json.dumps(...)) for writing to a file. Also, please call f.close() once you are finished with it